package.json

```diff
@@ -15,6 +15,7 @@
     "react-perfect-scrollbar": "^1.5.8",
     "react-router-dom": "^5.2.0",
     "react-scripts": "4.0.0",
+    "recharts": "^2.0.0-beta.8",
```

Comment:
@tams2429 Is there no stable version of this we can use, rather than `-beta`

Reply:
The most recent stable downgrade would be to version 1.8.5 as the version 2.0.0 has some issues, but haven't tested what the loss in functionality would be? Can try to test it? But for now managed to get it deployed with the beta version